tests/acceptance/poc-test.js

```diff
@@ -1,23 +0,0 @@
-import Ember from 'ember';
```

Comment:
I like the addition of node-side tests, I've been meaning to add them, but I would like to keep the ember tests as well. I use them as an acceptance test and for debugging ... within the project I can run `ember test` and see it run with each scenario, with my code changes without having to hunt for another project to use.